From 65ecf004cbcb03a978c436a0e04753e27fdb2ebe Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sun, 8 Oct 2023 10:56:29 +0200
Subject: [PATCH 12/12] Fix gtest pkg-config detection

For some reason now there's some issue when passing the version string
to pkg_check_modules with spaces inbetween which breaks setting a bunch
of used variables.

  pkg_check_modules(GTEST "gtest >= 1.8.0")

  -- Checking for module 'gtest >= 1.8.0'
  --   Found gtest , version

vs.

  pkg_check_modules(GTEST "gtest>=1.8.0")

  -- Checking for module 'gtest>=1.8.0'
  --   Found gtest, version 1.14.0
---
 cmake/FindGtestGmock.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/FindGtestGmock.cmake b/cmake/FindGtestGmock.cmake
index e0e3c9c1a3..507a543bbc 100644
--- a/cmake/FindGtestGmock.cmake
+++ b/cmake/FindGtestGmock.cmake
@@ -1,7 +1,7 @@
 include(FindPackageHandleStandardArgs)
 
-pkg_check_modules(GTEST QUIET "gtest >= 1.8.0")
-pkg_check_modules(GTEST_MAIN QUIET "gtest_main >= 1.8.0")
+pkg_check_modules(GTEST QUIET gtest>=1.8.0)
+pkg_check_modules(GTEST_MAIN QUIET gtest_main>=1.8.0)
 if (GTEST_FOUND AND GTEST_MAIN_FOUND)
     # If we can find package configuration for gtest use it!
     set(GTEST_LIBRARY ${GTEST_LIBRARIES})
-- 
2.42.0

